doc/source/whatsnew/v0.9.1.txt

```diff
@@ -89,12 +89,21 @@ New features
 
   - Enable referencing of Excel columns by their column names (:issue:`1936`)
 
-    .. ipython:: python
-
-        xl = ExcelFile('data/test.xls')
-        xl.parse('Sheet1', index_col=0, parse_dates=True,
-                 parse_cols='A:D')
+    .. code-block:: python
 
+       In [3]: xl = pd.ExcelFile('data/test.xls')
+
+       In [5]: xl.parse('Sheet1', index_col=0,
+                        parse_cols='A:D')
+       Out[5]:
+                          A         B         C
+       2000-01-03  0.980269  3.685731 -0.364217
+       2000-01-04  1.047916 -0.041232 -0.161812
+       2000-01-05  0.498581  0.731168 -0.537677
+       2000-01-06  1.120202  1.567621  0.003641
+       2000-01-07 -0.487094  0.571455 -1.611639
+       2000-01-10  0.836649  0.246462  0.588543
+       2000-01-11 -0.157161  1.340307  1.195778
```

Comment:
Can you leave this out? As it is IMO actually a regression that `parse_dates` does not work anymore (ref https://github.com/pydata/pandas/pull/12051)